web/beacon-app/src/features/onboarding/components/StepButtons.tsx

```diff
@@ -21,7 +29,7 @@ const StepButtons = ({ isSubmitting, isDisabled }: StepButtonsProps) => {
       <Button type="submit" isLoading={isSubmitting} disabled={isDisabled || isSubmitting}>
         <Trans>Next</Trans>
       </Button>
-      {currentStep !== ONBOARDING_STEPS.ORGANIZATION && (
+      {shouldDisplayBackButton && (
```

Comment:
If we don't want invited users to navigate back to steps 1 and 2, should they see `Step 3 of 4` at the top of the `Name` form? 

If that is changed, the stepper in the sidebar would also have to updated so that they only see 2 steps instead of 4.